gitlab/runpipe-tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+. gitlab/ci_settings.sh
+
+section_start compile "Compile runpipe"
+# Configure and make the runpipe binaries.
+make configure
+./configure --with-baseurl='http://localhost/domjudge/' --with-domjudge-user=domjudge --with-judgehost_chrootdir=${DIR}/chroot/domjudge |& tee "$GITLABARTIFACTS/configure.log"
+make judgehost |& tee "$GITLABARTIFACTS/make.log"
+section_end compile
+
+cd judge/test
```

Comment:
CI is complaining:
```
shellcheck: gitlab/runpipe-tests.sh:12:1: warning: Use 'cd ... || exit' or 'cd ... || return' in case cd fails. [SC2164]
```

So

```suggestion
cd judge/test || exit
```